Formula/kubernetes-cli.rb

```diff
@@ -27,7 +27,21 @@ def install
 
     dir = "_output/local/bin/darwin/#{arch}"
     bin.install "#{dir}/kubectl"
+
     (bash_completion/"kubectl").write Utils.popen_read("#{bin}/kubectl completion bash")
+    (zsh_completion/"_kubectl").write Utils.popen_read("#{bin}/kubectl completion zsh")
+  end
+
+  def caveats; <<-EOS.undent
+    Add the following to ~/.zshrc to enable zsh completion:
+      source #{HOMEBREW_PREFIX}/share/zsh/site-functions/_kubectl
```

Comment:
Right, but only if the script starts with the `#compdef` header.

Ref: http://zsh.sourceforge.net/Doc/Release/Completion-System.html

> When compinit is run, it searches all such files accessible via fpath/FPATH and reads the first line of each of them. This line should contain one of the tags described below. Files whose first line does not start with one of these tags are not considered to be part of the completion system and will not be treated specially.